src/components/table/table.tsx

```diff
@@ -69,7 +70,13 @@ export default function Table<T extends object, C extends TableConfig<T>>({
             ))}
           <tr>
             <td colSpan={columns.length}>
-              <styled.TableMessage>{endMessage}</styled.TableMessage>
+              <styled.TableMessage>
+                {endMessageProps.kind === 'infinite-scroll' ? (
+                  <TableInfiniteScrollLoader {...endMessageProps} />
+                ) : (
+                  <>{endMessageProps.content}</>
```

Comment:
The `endMessageProps.content` styles doesn't match styles of `TableInfiniteScrollLoader` text

Reply:
This is intentional, it's a React node so if there is no end message, we want to render `null` instead of an empty DOM element.